aplicaciones/academico/views.py

```
@@ -1,5 +1,7 @@
+from django.forms import ValidationError
+from rest_framework.viewsets import ModelViewSet 
```

Ruff (0.11.9)

`rest_framework.viewsets.ModelViewSet` imported but unused

Remove unused import: `rest_framework.viewsets.ModelViewSet`

(F401)